scalding-core/src/main/scala/com/twitter/package.scala

```diff
@@ -31,6 +32,8 @@ package object scalding {
   type ValuePipe[+T] = com.twitter.scalding.typed.ValuePipe[T]
   type Grouped[K, +V] = com.twitter.scalding.typed.Grouped[K, V]
 
+  implicit def typedDescriptor[T]: TypeDescriptor[T] = macro com.twitter.scalding.macros.impl.TypeDescriptorProviderImpl.caseClassTypeDescriptorImpl[T]
```

Comment:
I guess if we don't do this we can't replace TypedTsv, right?

What if we put this method in the `object TypedDescriptor`? I think then it would be found by default if not explicitly overridden by another import.